argus/action_manager/windows.py

```diff
@@ -177,11 +178,34 @@ def git_clone(self, repo_url, location):
             Specific location on the instance.
         """
         LOG.info("Cloning from %s to %s", repo_url, location)
-        cmd = "git clone {} {}".format(repo_url, location)
-        self._client.run_command_with_retry(cmd,
-                                            count=util.RETRY_COUNT,
-                                            delay=util.RETRY_DELAY,
-                                            command_type=util.CMD)
+        git_clone_cmd = "git clone {} {}".format(repo_url, location)
+        del_repo_cmd = ("If (Test-Path -PathType Container -Path \"{0}\") "
```

Comment:
Why you don't use `'` in order to avoid escaping.
For example:
```
'If (Test-Path -PathType Container -Path "%(location)s")'
    '{ Remove-Item -Recurse -Force "%(location)s" }' % {"location": location}